pkg/registry/networking/networkpolicy/strategy.go

```diff
@@ -44,9 +46,31 @@ func (networkPolicyStrategy) NamespaceScoped() bool {
 	return true
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (networkPolicyStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	fields := map[fieldpath.APIVersion]*fieldpath.Set{
+		"extensions/v1beta1": fieldpath.NewSet(
+			fieldpath.MakePathOrDie("status"),
+		),
```

Comment:
@thockin , for learnin, why do we need to change these API group?

Reply:
I learned this just recently - this method allows us to express "auto-clear status when spec is written" and vice-versa without doing that manually.